components/Tabs/tests/snapshot/TabBarView/MDCTabBarViewSnapshotTests.m

```diff
@@ -137,4 +137,63 @@ - (void)testItemsWithMixedTitlesAndImages {
   [self generateSnapshotAndVerifyForView:self.tabBarView];
 }
 
+- (void)testMultipleLineItemViewToFitInTheSingleScreen {
+  // Given
+  UITabBarItem *item1 = [[UITabBarItem alloc] initWithTitle:@"One" image:nil tag:0];
+  UITabBarItem *item2 = [[UITabBarItem alloc] initWithTitle:@"Two" image:nil tag:2];
+  UITabBarItem *item3 = [[UITabBarItem alloc] initWithTitle:@"Three" image:nil tag:5];
+  UITabBarItem *item4 = [[UITabBarItem alloc] initWithTitle:@"Four" image:nil tag:6];
+  UITabBarItem *item5 = [[UITabBarItem alloc] initWithTitle:@"Five" image:nil tag:7];
+  UITabBarItem *item6 = [[UITabBarItem alloc] initWithTitle:@"Six" image:nil tag:8];
+
+  // When
+  self.tabBarView.items = @[ item1, item2, item3, item4, item5, item6 ];
+
+  // Then
+  [self generateSnapshotAndVerifyForView:self.tabBarView];
+}
+
+- (void)testTooManyItemsInScrollView {
```

Comment:
Nit: Naming: Please remove "InScollView" from the test since the behavior is automatic and depends on bounds sizes, etc.  Consider the name `testItemsLargerThanBoundsChangesToScrollableLayout`?

Reply:
Thanks for the advising!